.depcheckrc.yml

```diff
@@ -1,12 +1,2 @@
-ignores: [
-  "@oclif/dev-cli",
-  "depcheck",
-  "eslint*",
-  "mocha",
-  "nyc",
-  "prettier",
-  "snyk",
-  "sort-package-json",
```

Comment:
``` 
~/src/raml-toolkit-will  wjh/download-cmd.W-7710840  0m  npm run depcheck                                                                      

> @commerce-apps/raml-toolkit@0.4.1 depcheck /Users/bredmond/src/raml-toolkit-will
> depcheck

Unused devDependencies
* sort-package-json
* tslib
```
The tslib one doesn't trigger after `npm run compile`, but the sort-package-json still does.

Reply:
Why does this pass on circle then?

Reply:
Hmmm, maybe we upgraded depcheck. I manually deleted my node_modules, ran npm install and now it's not complaining. I'm guessing that's why none of the other ignores are needed now as well.